lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -204,6 +209,11 @@ Function Choco-Upgrade
         $cmd += " -ignoredependencies"
```

Comment:
Probably something for the next Windows meeting, I don't want to make a decision here without a consensus